notifications_python_client/base.py

```diff
@@ -71,7 +72,7 @@ def request(self, method, url, data=None, params=None):
             self.service_id
         )
 
-        url = urlparse.urljoin(self.base_url, url)
+        url = urllib.parse.urljoin(str(self.base_url), str(url))
```

Comment:
urlparse is the python 3 version of urllib.parse - however the future thing patches that into old urllib rather than vice versa :(

unless nice about casting everything to `str`

i hate unicode